resources/cloudtrail_trails.go

```diff
@@ -275,14 +280,51 @@ func postCloudtrailTrailResolver(ctx context.Context, meta schema.ClientMeta, re
 	}
 	return nil
 }
+func resolveCloudtrailTrailTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
You can list 20 ARNS in ListTags why not do the resolve inside the main fetch, this is an easy performance win.

https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_ListTags.html